storehouse/stock.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Start functions for storehouse
+ */
+
+let carPartsInStock = ['alternator', 'klocki hamulcowe'];
+
+/**
+ * @param partName (string)
+ */
+function addToStock (partName) {
+    carPartsInStock.push(partName);
+}
+addToStock('silnik');
+
+function takeFromStock () {
```

Comment:
Ta funkcja ma dwie dolegliwości.
Pierwsza jest taka, że niepotrzebnie sprawdzasz czy tablica ma jakieś elementy.
Metoda shift uruchomi się nawet wtedy gdy tablica jest pusta.

Kolejna sprawa to nigdzie nie widze "return".
Ta funkcja literalnie usuwa pierwszy element z listy, zamiast go pobierać i zwracać.

Po uruchomieniu tej funkcji element znika, ale ja nie mogę z nim nic zrobić.